docs/source/quickstart-gcp.rst

```diff
@@ -58,21 +76,9 @@ Set up an output bucket (if one doesn't exist)
 Configure ElasticBLAST
 ----------------------
 
-The minimal configuration requires: 
-
-#. Cloud service provider configuration (see :ref:`GCP configuration <gcp_conf>` for details),
-
-#. :ref:`query sequences <elb_queries>`, 
-
-#. a :ref:`cloud storage bucket for results <elb_results>`. This value must start with ``gs://`` and _uniquely_ identifies your ElasticBLAST search. **Please keep track of this**.
+You will use a configuration file to specify your input to ElasticBLAST.  Once you have written the configuration file, you'll just need to tell ElasticBLAST about it when invoked.
 
-#. basic BLAST parameters (:ref:`program <elb_blast_program>` and :ref:`database <elb_db>`), and
-
-#. :ref:`elb_num_nodes` to start.
-
-
-
-They can be provided on a standard `ini configuration file format <https://en.wikipedia.org/wiki/INI_file>`_, e.g.:
+Start by copying the configuration file shown below.  Using an editor, write this text to a new file called "BDQA.ini".  Both nano and vi are available on the Cloud Shell.
```

Comment:
```suggestion
Start by copying the configuration file shown below.  Using an editor, write this text to a new file called "BDQA.ini".  Both `nano <https://www.nano-editor.org/>`_ and `vi <https://en.wikipedia.org/wiki/Vi>`_ are available on the Cloud Shell.
```

Reply:
Same comment as above.  If folks don't know what it is, they can google it.   The link is distracting.